lib/package.rb

```diff
@@ -164,13 +164,19 @@ def self.system(*args)
       { exception: true }
     end
 
-    # add -j arg to build commands
+    # Add -j arg to build commands.
     cmd_args.sub(/\b(?<=make)(?=\b)/, " -j#{CREW_NPROC}") unless cmd_args =~ /-j\s*\d+/
+    # Escape special bash characters.
+    cmd_args.gsub!('"','\\\\"')
+    cmd_args.gsub!('$','\\\\$')
+    cmd_args.gsub!('`','\\\\`')
```

Comment:
Seriously?

Reply:
Feel free to make this code better. This is at the "this works to fix the problem" stage. The next step is to optimize it...